js/samples/04.ai-apps/h.datasource-azureOpenAI/README.md

```diff
@@ -62,7 +62,7 @@ This sample shows how to integrate your Azure AI Search index as a data source i
 > [!NOTE]
 > Please note that at this time, this sample is only supported with Azure OpenAI.
 
-1. Fill the `AZURE_OPENAI_KEY`, `AZURE_OPENAI_ENDPOINT`, `AZURE_SEARCH_KEY`, `AZURE_SEARCH_ENDPOINT` variables appropriately.
+1. Fill the `AZURE_OPENAI_KEY`, `AZURE_OPENAI_ENDPOINT` variables appropriately.
```

Comment:
Please update lines 76 and 80 of this file to match your changes to config.json

Reply:
Updated, thanks Corina.